__tests__/api.test.js

```diff
@@ -483,3 +483,32 @@ describe("PATCH /api/articles/:article_id", () => {
 			});
 	});
 });
+
+describe("DELETE /api/comments/:comment_id", () => {
+	test("STATUS 200: Shoud delete a given comment and respond with status 204 and no content.", () => {
+		return request(app)
+			.delete("/api/comments/13")
+			.expect(204)
```

Comment:
It would be fine to just have this expect without checking the response, as the response is empty with a 204